doc/news/_preparation_next_release.md

```diff
@@ -58,6 +58,7 @@ please take a look at the [README](https://www.libelektra.org/plugins/type).
 ## Plugins
 
 The following section lists news about the [modules](https://www.libelektra.org/plugins/readme) we updated in this release.
+- Support DOS newlines for the csvstorage plugin. _(Vlad - Ioan Balan)_
```

Comment:
Seems like you need to reformat this file with [Prettier](https://www.prettier.io), otherwise the test [`check_formatting`](https://github.com/ElektraInitiative/libelektra/blob/master/tests/shell/check_formatting.sh) will fail. You can also [use the instructions printed by the Jenkins build server](https://build.libelektra.org/jenkins/blue/organizations/jenkins/libelektra/detail/PR-2553/4/pipeline/223/) to fix the problem, if you do not want to install `prettier`.